src/stan-users-guide/latent-discrete.Rmd

```diff
@@ -1019,3 +1019,182 @@ the probability mass function of $z_i$ in the posterior.   These may
 be averaged across the iterations to provide the posterior probability
 distribution over each $z_i$.
 
+## The mathematics of recovering marginalized parameters {#marginalization-mathematics.section}
+
+### Introduction {-}
+
+In this section we describe in more detail the mathematics of statistical 
+inference using the output of marginalized Stan models, such as those presented 
+in the last three sections. This section is intended to provide a mathematical
+explanation of why and how certain manipulations of Stan's output produce valid
+summaries of the posterior distribution when discrete parameters have been
+marginalized out of a statistical model. Ultimately, however, fully understanding
+the mathematics in this section is **not** necessary to fit models with discrete
+parameters using Stan.  
+
+Throughout we will work with a model consisting of both continuous parameters, 
+$X$, and discrete parameters, $Y$. We will also assume that $Y$ can only take
+finitely many values, as is the case for all the models described in this 
+chapter of the User's Guide. In order to simplify notation, we suppress any
+conditioning on data in this section, except where specified. As with all 
+Bayesian analyses, however, all inferences using models with marginalized
+parameters are made conditional on the observed data. 
+
+### Estimating expectations {-}
+
+When making Bayesian inferences, we are generally interested in estimating some 
+quantity of interest $\mu = \textsf{E}(g(X, Y))$ where $g(\cdot)$ is an arbitrary 
+function. For example, we may be interested in $\textsf{E}(X)$, the posterior 
+mean of the continuous parameters, where we would take $g(x, y) = x$. In this
+context, by *estimating*, we mean estimating in the *Monte Carlo* sense, which
```

Comment:
The purpose of this paragraph is to clarify the usage of the words 'estimating', 'estimation', etc. in this whole section, in case people misinterpret it.  But perhaps this is still a bit confusing.

Would it help to use the language of 'calculating' rather than 'estimating'?

That is, we are **calculating** posterior quantities, in this case via a Monte Carlo method.  Those quantities (once calculated accurately) are later used for inference (i.e. for interpreting the evidence in the context of the analysis), but we don't discuss any of those later steps here.  We are purely discussing how to go about calculating various quantities.